12_12_2019_HW.py

```diff
@@ -0,0 +1,105 @@
+import requests
+import random
+import string
+
+from flask import Flask
+from flask import request
+from faker import Faker
+
+from data_base import exec_query
+
+app = Flask('app')
+
+
+@app.route('/req')
+def requirements():
+    with open('requirements.txt') as f:
+        return '<br>'.join(row for row in f.read().split('\n'))
```

Comment:
'<br>'.join(row for row in f.read().split('\n'))
f.read().replace('\n', '<br>')